File: backend/src/main/java/be/pdp/modelcar/dto/CarDto.java
package be.pdp.modelcar.dto;

import lombok.Getter;
import lombok.Setter;

/**
 * Created by peterdp on 27/10/2017.
 */
@Getter @Setter
public class CarDto extends Dto {

    private String itemRef;

    private ModelDto modelDto;

    private ColorDto colorDto;

    private int limitedEdition;

    private boolean coupe;

    private boolean boxed;

    private ScaleDto scale;


}

File: modelcar/src/app/domain/CarDto.ts
import {BrandDto} from './BrandDto';
import {ColorDto} from './ColorDto';
import {ModelDto} from './ModelDto';
import {ScaleDto} from './ScaleDto';

export class CarDto {
  itemRef: string;
  modelDto: ModelDto;
  colorDto: ColorDto;
  limitedEdition: number;
  coupe: boolean;
  boxed: boolean;
  scale: ScaleDto;

  constructor() {

  }

}


